docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md

```diff
@@ -103,7 +103,7 @@ ToResponseMarshaller.oneOf(
 }
 ```
 
-After:
+After: 
```

Comment:
Uhm, ok... I see.
I mean, with Pull Requests in open source projects in general a PR is usually for adding the missing/new content, not just commenting what needs to be done. After all, we're an open source project and rely on the communities participation in its development :-)

If you don't want to contribute this section that's too bad, I hoped you'd help us after we helped you on the mailing list... :-) In any case, shall I close this PR or are you considering writing something up still?

Thanks in any case and happy hakking 🎄